.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: node_js
 
-node_js:
-  - 8
-  - 10
+env:
+  - FORCE_COLOR=true
 
-branches:
-  only: master
+node_js:
+  - 10.12
+  - 12
+  - node
 
 jobs:
   include:
     - stage: release
+      if: branch = master AND type != pull_request
```

Comment:
```suggestion
      if: branch = master AND type = push
```

What do you think?

Reply:
Mmm I'm not sure. There are other types which I'm not sure if we are interested to trigger or not. Ignoring pull requests is the usual behavior I've seen in other packages.